ratis-client/src/main/java/org/apache/ratis/client/impl/OrderedStreamAsync.java

```diff
@@ -147,16 +153,16 @@ private void sendRequestToNetwork(DataStreamWindowRequest request,
 
     scheduleWithTimeout(request);
 
-    requestFuture.thenApply(reply -> {
+    requestFuture.thenApplyAsync(reply -> {
```

Comment:
This is only for processing the reply which is simple memory operations without any IO.  Are you sure this would help the performance?

Since we are using Netty, there is already a EventLoopGroup in Netty to do the send IO; see https://github.com/apache/ratis/blob/d41d0930c49c4cfcd8cd709519329e22fbbe61b4/ratis-netty/src/main/java/org/apache/ratis/netty/client/NettyClientStreamRpc.java#L247-L259